File: Java/src/ValidParenthesesTest.java
import org.junit.Assert;
import org.junit.Test;

public class ValidParenthesesTest {

    @Test
    public void testsValid() {
        ValidParentheses vp = new ValidParentheses();

        Assert.assertTrue(vp.isValid("()"));

        Assert.assertTrue(vp.isValid("()[]{}"));

        Assert.assertFalse(vp.isValid("("));

        Assert.assertFalse(vp.isValid("(]"));

        Assert.assertFalse(vp.isValid("([)]"));
    }

}

File: Kotlin/src/ValidParenthesesTest.kt
import org.junit.Test
import kotlin.test.assertFalse
import kotlin.test.assertTrue

class ValidParenthesesTest {

    @Test
    fun testIsValid() {
        val vp = ValidParentheses()

        assertTrue(vp.isValid("()"))

        assertTrue(vp.isValid("()[]{}"))

        assertFalse(vp.isValid("("))

        assertFalse(vp.isValid("(]"))

        assertFalse(vp.isValid("([)]"))
    }

}